frontend/src/helpers/policies.ts

```diff
@@ -0,0 +1,12 @@
+import { PolicyType } from "@app/hooks/api/policies/enums";
+
```

Comment:
We are trying to move away from all these helpers things because we want to follow co-location principle. 

Its better to keep it in there places that keeping it here

Reply:
There is not a clear place where to define this shared enums. I think we should follow the current approach until we define a better one.